clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -45,21 +80,7 @@ fn suggest_long_flag_subset() {
                 .action(clap::ArgAction::Count),
         );
 
-    let args = [name, "--he"];
-    let arg_index = 1;
-    let args = IntoIterator::into_iter(args)
-        .map(std::ffi::OsString::from)
-        .collect::<Vec<_>>();
-    let current_dir = None;
-
-    let completions =
-        clap_complete::dynamic::complete(&mut cmd, args, arg_index, current_dir).unwrap();
-    let completions = completions
-        .into_iter()
-        .map(|s| s.0.to_string_lossy().into_owned())
-        .collect::<Vec<_>>();
-
-    assert_eq!(completions, ["--hello-world", "--hello-moon", "--help"]);
+    assert_completion!(cmd, "--he", ["--hello-world", "--hello-moon", "--help" => HELP_SHORT]);
 }
```

Comment:
I wrote it as a macro now, but I imagine, using a string would allow using snapshots, so that could be preferable. what do you think @epage.